proposed/http-client/http-client-meta.md

```diff
@@ -38,24 +38,24 @@ specify the default behaviours.
 
 ### Default behavior
 
-The intention of this PSR is ensure library developers that all HTTP clients have the same 
-**default behavior**. That means that all HTTP clients MUST follow Liskov substitution principle
-when no configuration is provided. The PSR does not try to restrict nor define configuration for 
-HTTP clients. An implementing library is free to be configured by the application author to follow
-redirects, to throw exceptions or any other possible setting.  
+The intention of this PSR is to provide library developers with HTTP clients that 
+have a well defined behaviour. A library should be able to use any compliant client
+without special code to handle client implementation details (Liskov substitution
+principle). The PSR does not try to restrict nor define configuration interface 
```

Comment:
`... define configuration interface for ...` is grammatically not correct. it could be `... define a configuration interface for ...` or `... define how to configure ...`. i would prefer the later, we do not restrict or define how to configure HTTP clients.